client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/dialog-styles.scss

```diff
@@ -1,8 +1,8 @@
 .carrier-accounts__settings-cancel-dialog {
 	.dialog__content {
 		padding: 0;
-		width: 666px;
-		height: 200px;
+		max-width: 666px;
```

Comment:
The ticket stated:
> we can remove the `width: 666px` and instead use a variable width to ensure visibility of text and exit icon on smaller screen sizes

However By doing that, the modal would have expanded considerably. I changed to `max-width` so that it doesn't look _as_ wide on lg+ screens